src/test/java/github/weichware10/util/config/ConfigurationTest.java

```diff
@@ -0,0 +1,23 @@
+package github.weichware10.util.config;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/**
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.